pkg/cli/mirror/mirror.go

```diff
@@ -290,284 +290,285 @@ func (o *MirrorOptions) Run(cmd *cobra.Command, f kcmdutil.Factory) (err error)
 	diskToMirror := len(o.ToMirror) > 0 && len(o.From) > 0
 	mirrorToMirror := len(o.ToMirror) > 0 && len(o.ConfigPath) > 0
 
-	if o.UseOCIFeature {
```

Comment:
o.EI don't think this improves the situation much, I'd suggest something like:
```golang
if o.UseOCIFeature {
    return mirrorOCIImage()
}
return mirrorImage()
```
then copy the contents of those if blocks under those functions and it'll be so much easier to read.

Reply:
OK this explains it better. thanks

Reply:
Change done. Thank you @soltysh